### ISSUE TITLE
A11y_CustomerServiceInsights_Share Workspace - Share_AI4W: MAS 4.1.1,MAS 4.1.2: ARIA attributes must conform to valid values (.ms-BasePicker-input)

### ISSUE THREAD

**Issue by Vividha-Khandelwal** [OfficeDev/office-ui-fabric-react#10615]
### Environment Information

- **Package version(s)**: (fill this out)
- **Browser and OS versions**: Chrome Version 77.0.3865.90 (Official Build) (64-bit)

> Note that our screen reader support is limited to Edge + Narrator. Issues that do not reproduce in Edge + Narrator, and aren't caused by obvious invalid aria values, should be filed with the respective screen reading software, not the Fabric repo.

### Describe the issue:

Issue
ARIA attributes must conform to valid values (aria-valid-attr-value) 

![5250722_Share Workspace - Share_AI4W](https://user-images.githubusercontent.com/48586069/65586580-026e7480-dfa2-11e9-86fa-85eeeae9cf5a.png)

Element path
.ms-BasePicker-input 

Snippet
<input spellcheck="false" aria-label="Email list for sharing" placeholder="Enter email address" class="ms-BasePicker-input input-334" aria-expanded="true" aria-haspopup="true" autocomplete="off" role="combobox" aria-controls="suggestion-list-id__3002 " …

How to fix
Fix all of the following:
- Invalid ARIA attribute values: aria-controls="suggestion-list-id__3002 " aria-owns="suggestion-list-id__3002"

**Comment by joschect** [comment]
@Vividha-Khandelwal could you provide a [codepen](https://codepen.io/slimemold/pen/qBWweNz?&editable=true) that reproduces this? I do not see this same behavior in the current deployed picker bits. Additionally, what version of fabric are you on?

![image](https://user-images.githubusercontent.com/20801176/65789765-e9d19c00-e112-11e9-8611-35365ded6054.png)

**Comment by easy0924** [comment]
So you can just find this issue mentioned in the official UI Fabrics document examples:
https://developer.microsoft.com/en-us/fabric#/controls/web/peoplepicker

Issues occurs when you type a search term that returns no results, as following:
![image](https://user-images.githubusercontent.com/2107890/66156192-0b7cc880-e5d6-11e9-81a8-1e636cbb05de.png)

Note in the HTML, the aria-controls and aria-owns value of that input points to an element Id that doesn't exist (suggestion-list-id__185 in this example)
![image](https://user-images.githubusercontent.com/2107890/66156268-3404c280-e5d6-11e9-8196-c429b80ddf8c.png)

So that's the accessibility issue.

**Comment by joschect** [comment]
@easy0924 Thanks for clarifying what you meant. I can see now that reproduces. We'll get this fixed.

**Comment by joschect** [comment]
This was fixed in #10872
